### PR TITLE
CSI-1882 / add parameters to jenkinsfile

### PR DESCRIPTION
**Pull request by matancarmeli7**

### CODE REVIEW

**Review by zingero — APPROVED** [review]

**Review by barakda — DISMISSED** [review]

File: scripts/ci/jenkins_pipeline_csi [hunk 1]
```diff
@@ -1,4 +1,13 @@
 pipeline {
+    parameters {
+        string(name: 'IMAGE_VERSION', defaultValue: "1.7.0")
+        string(name: 'CSI_CONTROLLER_IMAGE', defaultValue: "default")
+        string(name: 'CSI_NODE_IMAGE', defaultValue: "ibm-block-csi-driver-node-amd64")
+        string(name: 'DOCKER_REGISTRY', defaultValue: "stg-artifactory.haifa.ibm.com:5030")
+        string(name: 'EMAIL_TO', defaultValue: "")
+        string(name: 'BANDIT_SKIPS', defaultValue: "xavi-system-tests", description: "comma separated list of issues to skip from the bandit python security test.\nB506: https://jira.xiv.ibm.com/browse/CSI-500")
```
Comment (oriyarde) [inline scripts/ci/jenkins_pipeline_csi:8]: how is `xavi-system-tests` relevant to the driver?
CSI-500 was [already done](https://github.com/IBM/ibm-block-csi-driver/pull/90)

Comment (matancarmeli7) [reply scripts/ci/jenkins_pipeline_csi]: done

Comment (oriyarde) [reply scripts/ci/jenkins_pipeline_csi]: I meant that I don't think we should keep the resolved `B506` skip.
and I couldn't find bandit anywhere in this project, so I think we can remove the parameter altogether

Comment (oriyarde) [reply scripts/ci/jenkins_pipeline_csi]: the only place where I think this parameter could be used, is a different job named `CSI_code_scanning`, maybe

**Review by zingero — DISMISSED** [review]

File: scripts/ci/jenkins_pipeline_csi (hunk 1 above)
Comment (matancarmeli7) [reply scripts/ci/jenkins_pipeline_csi]: so why this parameter is still in the Jenkins jobs with this description?

Comment (matancarmeli7) [reply scripts/ci/jenkins_pipeline_csi]: removed them

Comment (oriyarde) [reply scripts/ci/jenkins_pipeline_csi]: > so why this parameter is still in the Jenkins jobs with this description?

legacy I guess

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**